test/e2e_tests/specs/ClearConversationContent/clearConversationContent.spec.ts

```diff
@@ -0,0 +1,400 @@
+    {tag: ['@TC-156', '@regression']},
+    async ({createPage}) => {
+      const [userAPageManager, userBPageManager, userCPageManager] = await Promise.all([
+        PageManager.from(createPage(withLogin(userA), withConnectedUser(userB), withConnectedUser(userC))),
+        PageManager.from(createPage(withLogin(userB))),
+        PageManager.from(createPage(withLogin(userC))),
+      ]);
+
+      const {pages: userAPages, modals: userAModals} = userAPageManager.webapp;
+      const userBPages = userBPageManager.webapp.pages;
+      const userCPages = userCPageManager.webapp.pages;
+
+      // Step 1: Create a group conversation with User A, B and C
+      const conversationName = 'Group conversation';
+      await createGroup(userAPages, conversationName, [userB, userC]);
+
+      // Step 2: Write messages in the group conversation
+      await userAPages.conversationList().openConversation(conversationName);
+      await userAPages.conversation().sendMessage('Message from User A');
+
+      await userBPages.conversationList().openConversation(conversationName);
+      await userBPages.conversation().sendMessage('Message from User B');
+
+      await userCPages.conversationList().openConversation(conversationName);
+      await userCPages.conversation().sendMessage('Message from User C');
+
+      // Step 3: User A selects 'Clear Conversation' option from the Conversation List Context Menu
+      await userAPages.conversationList().openContextMenu(conversationName);
+      await userAPages.conversationList().clearContentButton.click();
+      // Step 4: Warning Popup should open
+      await expect(userAModals.optionModal().modal).toBeVisible();
+      // Step 5: User A clicks 'Clear'
+      await userAModals.optionModal().clickAction();
+
+      // Step 6: Verify you can receive incoming new messages, pings, calls, pictures, and files
+      // Step 6.1: Verify you can receive incoming new messages
+      await userBPages.conversationList().openConversation(conversationName);
+      await userBPages.conversation().sendMessage('Message from User B after Clear');
+      await userAPages.conversationList().openConversation(conversationName);
+      await expect(userAPages.conversation().messages).toHaveCount(1);
+
+      // Step 6.2: Verify you can receive incoming pings
+      await userBPages.conversationList().openConversation(conversationName);
+      await userBPages.conversation().sendPing();
+      await userAPages.conversationList().openConversation(conversationName);
+      await expect(userAPages.conversation().getPing()).toBeVisible();
+
+      // Step 6.3: Verify you can receive incoming pictures
+      const {page} = userAPages.conversation();
+      await shareAssetHelper(getImageFilePath(), page, page.getByRole('button', {name: 'Add picture'}));
+      await shareAssetHelper(
+        getImageFilePath(),
+        userBPageManager.page,
+        userBPageManager.page.getByRole('button', {name: 'Add picture'}),
+      );
```

Comment:
Why do you send a picture from A to B and one from B to A? There's only one assertion that A received the image from B, so sending one should be enough. Also please check your assertion on it. You only look for a message from B to be visible, this could also be the text message you sent earlier which might make your test pass even though the image isn't visible.